amun/api_v1.py

```diff
@@ -142,39 +159,69 @@ def post_inspection(specification: dict) -> tuple:
         run_memory_requests = specification['run']['requests']['memory']
 
     parameters['AMUN_INSPECTION_ID'] = inspection_id
+
     # Mark this for later use - in get_inspection_specification().
     specification["@created"] = datetime2datetime_str()
 
     # Create buildconfig, extend parameters with specification and generated dockerfile for build.
-    build_parameters = dict(parameters)
-    build_parameters['AMUN_GENERATED_DOCKERFILE'] = dockerfile
-    build_parameters['AMUN_SPECIFICATION'] = json.dumps(specification)
-    build_parameters['AMUN_CPU'] = specification['build'].get('requests', {}).get('cpu')
-    build_parameters['AMUN_MEMORY'] = specification['build'].get('requests', {}).get('memory')
-    _OPENSHIFT.schedule_inspection_build(build_parameters, inspection_id, use_hw_template)
-
-    if run_job:
-        _OPENSHIFT.schedule_inspection_job(
-            inspection_id,
-            parameters,
-            use_hw_template=use_hw_template,
-            memory_requests=run_memory_requests,
-            cpu_requests=run_cpu_requests
-        )
+    template_parameters = dict(parameters)
+    template_parameters['AMUN_CPU'] = specification['build'].get(
+        'requests', {}).get('cpu')
+    template_parameters['AMUN_MEMORY'] = specification['build'].get(
+        'requests', {}).get('memory')
```

Comment:
Please rebase, this has changed in the master.

Reply:
This is a subject of the #265 PR, I will rebase it there.